src/AllocatorBuffer.sol

```diff
@@ -61,17 +63,18 @@ contract AllocatorBuffer {
 
     // --- functions ---
 
-    function approve(
-        address token,
-        address spender,
-        uint256 amount
-    ) external auth {
-        TokenLike(token).approve(spender, amount);
-        emit Approve(token, spender, amount);
+    function approve(address asset, address spender, uint256 amount) external auth {
+        TokenLike(asset).approve(spender, amount);
+        emit Approve(asset, spender, amount);
     }
 
-    function deposit(address token, uint256 amount, address /* owner */) external {
-        TokenLike(token).transferFrom(msg.sender, address(this), amount);
-        emit Deposit(token, msg.sender, amount);
+    function deposit(bytes32 /*ilk*/, address asset, uint256 amount, address /* owner */) external {
```

Comment:
Where does this final parameter `owner` come from?

Reply:
Yes was actually wondering to ask you what was this about, need to dig a bit deeper.

Reply:
I've proceed to remove it as the `IAllocatorConduit` interface doesn't have it so has to be something that changed.